src/filter.jl

```diff
@@ -5,7 +5,7 @@ mypack(a::Number...) = [a...]
 mypack(a::SArray,b::SArray,c::Number) = ArrayPartition(a,b,@SVector[c])
 mypack(a::SArray,b::SArray,c::SArray) = ArrayPartition(a,b,c)
 
-compute_dP(B,P,σtil::Number) = B*P + P*B' .- outer_(σtil)
+compute_dP(B,P,σtil::Number) = B*P + P*B' .- σtil*σtil'
```

Comment:
That seems wrong, shouldn't it be
```suggestion
compute_dP(B,P,σtil::Number) = B*P + P*B' .- σtil*σtil'*I
```

Reply:
That results in some errors. the outer from above didn't work because `dP` was not a number anymore but a vector.

Reply:
Try 
```suggestion
compute_dP(B,P,σtil::Number) = B*P + P*B' - σtil*σtil'*I
```
as `I` doesn't support `.-`